test/evergreen.yml

```diff
@@ -45,6 +45,12 @@ functions:
     params:
       script: |
         git clone https://github.com/wiredtiger/mongo-tests
+  "fetch mongodb bin" :
+    command: shell.exec
+    params:
+      script: |
+        wget https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-5.0.0-rc4.tgz
```

Comment:
This is temporary. Building mongoDB takes too long, we could rely on available binaries. The best would be having the latest binary from a nightly build.

Reply:
I don't like the extra overhead of building mongo binaries, neither a temporary URL. Is it possible to find a more future-proof way to achieving it. The SDP team may have a better idea of what the nightly build URL looks like.

Reply:
We are going to have to do a build since we will provide a patch run on the latest MongoDB and custom branch (like for Jenkins Perf).